server/utils/generateEformReportValidation.ts

```diff
@@ -0,0 +1,77 @@
+import Joi from 'joi'
```

Comment:
Please can we delete this file. As we already have `generateReportValidation.ts`